README.md

```diff
@@ -1,19 +1,137 @@
 # elastalert-k8s-automation
-Use power of Kubernetes to automation configuration of elastaler
+Use power of Kubernetes to automation configuration of ElastAlert.
 
-## Building
-To build and upload image to the repository run the following commands:
+The goal of the current project is to simplify configuring ElastAlert notifications about events of some other 
+applications that store its logs in ElasticSearch.
+
+There are two modes for running this application: the local mode and the remote mode.
+* Remote mode is intended for running application in Kubernetes cluster where ElastAlert 
+is started.
+* Local mode allows to run application without deployment in Kubernetes. 
+
+The application uses two types of configuration files to configure work of ElastAlert:
+* Administrator configuration file with general properties and notification options.
+* User rules configuration files that determine rules for ElastAlert.
+
+The application collects both types of configuration files and generates configuration for ElastAlert.
+
+## Installing and configuring
+Clone elastlert-k8s-automation project:
+````
+$ git clone https://github.com/rvadim/elastalert-k8s-automation.git
+````
+
+Application can be configured by setting some environment variables:
+* `LOCAL_RUN`: if specified by value `1`, the Local mode will be chosen. 
+Otherwise the Remote mode wiil be chosen by default. (Optional)
+* `CONFIG_DIR`: if specified, the directory to administrator configuration of this application will be 
+specified by value of this variable. 
+Otherwise configuration file will be taken from "./eka/" directory of this project. (Optional)
+* `ELASTALERT_CONFIG_DIR`: if specified, the directory where generated ElastAlert configuration file will be placed. 
+Otherwise generated ElastAlert configuration file will be placed in "./config/" directory of this project. (Optional)
+
+*Note: it is assumed that ElastAlert and ElasticSearch are started*
+
+### Administrator configuration file
+The administrator configuration file must contain main options for ElastAlert and must have following structure:
+```yaml
+es_host: # Address of an ElasticSearch cluster
+
+es_port: # Port for es_host
+
+rules_folder: # Directory with ElastAlert rules
```

Comment:
This options not required, due to it calculated. Also, changing this parameter may cause fail to start.